libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -527,26 +527,40 @@ void AC_AutoTune::control_attitude()
 
         float target_max_rate;
         switch (axis) {
-        case ROLL:
+        case ROLL: {
             target_max_rate = MAX(AUTOTUNE_TARGET_MIN_RATE_RLLPIT_CDS, step_scaler*AUTOTUNE_TARGET_RATE_RLLPIT_CDS);
+            const float rll_rate_max_cds = degrees(attitude_control->get_ang_vel_roll_max_rads()) * 100;
```

Comment:
for naming consistency we should use roll_rate_max_cds

Reply:
done